logos/src/logos/capacity/capacity_planner.py

```
@@ -36,6 +36,7 @@
 )
 
 from .demand_tracker import DemandTracker
+from .lane_comparator import best_lane, lane_sort_key
```

Flake8 (7.3.0)

[error] '.lane_comparator.lane_sort_key' imported but unused

(F401)